.github/workflows/grain.yml

```diff
@@ -17,6 +14,12 @@ jobs:
         with:
           name: "grain"
           url: "https://github.com/grain-lang/grain/releases/download/grain-v0.3.2/grain-linux-x64"
+      - uses: engineerd/configurator@v0.0.7
+        with:
+          name: "wasmtime"
+          url: "https://github.com/bytecodealliance/wasmtime/releases/download/v0.27.0/wasmtime-v0.27.0-x86_64-linux.tar.xz?.tar.gz"
```

Comment:
Because it's an archive, you also need to add the relative path in the archive of the tool you need to add to the path:

`pathInArchive: wasmtime`